perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -24,6 +24,8 @@
 	/*#
 	 * Creates a subgroup of a group.
 	 *
+	 * Group object must contain name. Description and parentGroupId are optional. Other parameters are ignored.
```

Comment:
We should state, that ``parentGroupId`` param of ``group`` object is ignored in this case, since we use ``parentGroup`` param of the method itself to resolve actual parent group.